CONTRIBUTING.md

```diff
@@ -48,92 +52,142 @@ When you introduce a new whitelist comment, please don't forget to update the [w
 
 # Unit Testing
 
-TL;DR
+## Pre-requisites
+* WordPress-Coding-Standards
+* PHP CodeSniffer 2.9.x or 3.x
+* PHPUnit 4.x or 5.x
 
-If you have installed `phpcs` and the WordPress-Coding-Standards as [noted in the README](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards#how-to-use-this), then you can navigate to the directory where the `phpcs` repo is checked out and do:
+The WordPress Coding Standards use the PHP CodeSniffer native unit test suite for unit testing the sniffs.
 
-```sh
-composer install
-vendor/bin/phpunit --filter WordPress tests/AllTests.php
+Presuming you have installed PHP CodeSniffer and the WordPress-Coding-Standards as [noted in the README](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards#how-to-use-this), all you need now is `PHPUnit`.
+
+If you already have PHPUnit installed on your system: Congrats, you're all set.
+
+If not, you can navigate to the directory where the `PHP_CodeSniffer` repo is checked out and do `composer install` to install the `dev` dependencies.
+Alternatively, you can [install PHPUnit](https://phpunit.de/manual/5.7/en/installation.html) as a PHAR file.
+
+**Pro-tip**: PHPUnit is easiest to work with if you add the installation directory to your `PATH` environment variable.
```

Comment:
@GaryJones I've already remove the pro-tip part.

Other than that, I think how to handle dependencies and what dependencies are, is a different discussion. IMO there is a difference between application dependencies and dev tools, but that's for another day (while sitting in the :sun_with_face:  somewhere with a glass of :beer: or :wine_glass: )